e2e-tests/cypress/tests/integration/channels/accessibility/accessibility_sidebar_spec.ts

```diff
@@ -193,50 +194,48 @@ describe('Accessibility tests for RHS getting focus after buttons actions', () =
 
     it('Focus should be on RHS when opening Recent Mentions', () => {
         // # Click the Recent Mentions button
-        cy.findByRole('button', {name: /Recent mentions/i}).click();
+        cy.findByRole('button', {name: /Recent mentions/i}).focus().realPress('Enter');
```

Comment:
I believed click action was a workaround in the past and changing to focus+keypress/enter is a way to go with accessibility test. Just requesting to update the comment in code accordingly (to change "click"). Same request to take effect to other comments in code.

Reply:
done!